microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -10,35 +10,34 @@ keywords: microsoft edge, web development, f12 tools, devtools
 ---
 # Dark or light color scheme simulation  
 
-Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system is grating and can be an accessibility issue for some users.  
+Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system can be hard to read and can be an accessibility issue for some users.  
 
 For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS Media Query to detect whether the user prefers to display your product in a darker or lighter color scheme.  Then to test the result, use [Microsoft Edge DevTools][DevtoolsIndex] to simulate a change from dark to light mode, without having to change the setting for your entire operating system.  
 
 **To emulate dark or light theme:**
 
-1.  Open the **Command Menu**.  
-    1.  Select `Ctrl`+`Shift`+`P` \(Windows/Linux\) or `Command`+`Shift`+`P` \(macOS\).  
-        
-        :::image type="complex" source="../media/css-console-command-menu-rendering.msft.png" alt-text="The Command Menu" lightbox="../media/css-console-command-menu-rendering.msft.png":::
-           The **Command Menu**  
-        :::image-end:::  
-        
-1.  Type `emulate color`, choose either **Emulate CSS prefers-color-scheme: dark** or **Emulate CSS prefers-color-scheme: light** and then select `Enter`.  
+1.  Open the **Command Menu** by selecting `Ctrl`+`Shift`+`P` \(Windows/Linux\) or `Command`+`Shift`+`P` \(macOS\).  
+
+    :::image type="complex" source="../media/css-console-command-menu-rendering.msft.png" alt-text="The Command Menu" lightbox="../media/css-console-command-menu-rendering.msft.png":::
+        The **Command Menu**  
+    :::image-end:::  
+
+1.  Type "emulate color", choose either **Emulate CSS prefers-color-scheme: dark** or **Emulate CSS prefers-color-scheme: light**, and then select **Enter**.  
     
     :::image type="complex" source="../media/css-elements-styles-qs-select-renderingmode-command-menu.msft.png" alt-text="Color scheme option from Command Menu" lightbox="../media/css-elements-styles-qs-select-renderingmode-command-menu.msft.png":::
        Color scheme option from **Command** Menu  
     :::image-end:::  
     
     > [!IMPORTANT]
-    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeDarkTheme].  If you are wondering what to choose, make sure that you are choosing a **rendering** menu item, not an **appearance** menu item.  
+    > Simply typing `dark` or `light` does not reveal the right tool, since there is also a way to [choose a theme for DevTools][DevtoolsCustomizeTheme].  Select a **Rendering** command, rather than an **Appearance** command.  
```

Comment:
```suggestion
    > Select a **Rendering** command rather than an **Appearance** command.  The **Rendering** commands affect the rendered webpage.  The **Appearance** commands instead affect the DevTools part of the window, as described in [Apply color themes to Microsoft Edge DevTools][DevtoolsCustomizeTheme].
```